docs/detections/prebuilt-rules/tune-rule-signals.asciidoc

```diff
@@ -153,6 +153,7 @@ Take the following steps to tune indicator match rules:
 
 * Specify a detailed query as part of the indicator index query. Results of the indicator index query are used by the detection engine to query the indices specified in your rule definition's index pattern. Using no query or the wildcard `***` query may result in your rule executing very large queries.
 * Limit your rule's additional look-back time to as short a duration as possible, and no more than 24 hours.
+* Avoid cluster performance issues by scheduling your rule to run in one-hour intervals or longer. For example, avoid scheduling an indicator match rule to check for indicators every five minutes.
 
 NOTE: {es-sec} provides limited support for indicator match rules. See <<support-indicator-rules, support limitations>> for more information.
```

Comment:
Just pointing out these things as I see them. Your choice of replacement for "see". 
```suggestion
NOTE: {elastic-sec} provides limited support for indicator match rules. Visit <<support-indicator-rules, support limitations>> for more information.
```